js_modules/dagster-ui/packages/ui-core/tsconfig.workers.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "paths": {
+      "shared/*": ["./src/*"]
+    },
+    "target": "es2022",
+    "module": "es2022",
+    "strict": true,
+    "sourceMap": true,
+    "allowJs": true,
+    "jsx": "preserve",
+    "moduleResolution": "node",
+    "forceConsistentCasingInFileNames": true,
+    "noImplicitReturns": true,
+    "noUncheckedIndexedAccess": true,
+    "allowUnreachableCode": true,
+    "allowSyntheticDefaultImports": true,
+    "types": ["jest", "node"],
+    "skipLibCheck": false,
+    "lib": ["es2022", "webworker"],
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "noFallthroughCasesInSwitch": true
+  },
+  "include": ["src/workers/*.worker.ts", "worker-typings.d.ts"]
```

Comment:
This necessitates that all future workers also live in the `workers` directory.

Reply:
could we do `src/**/*.worker.ts` instead?